pallets/parachain-staking/src/lib.rs

```diff
@@ -1187,6 +1189,18 @@ pub mod pallet {
 
 	#[pallet::call]
 	impl<T: Config> Pallet<T> {
+		#[pallet::weight(0)]
+		pub fn unreserve_stakers(
```

Comment:
It might help us keep things straight if we name this something that implies it's a temporary fix, e.g. `hotfix_unreserve_staker` (or whatever makes sense). Same with the `UnreservedStaker` event.